handlers.py

```diff
@@ -30,26 +30,33 @@ def __init__(self, definition_repo):
         super().__init__()
         self.definitionRepo = definition_repo
 
+    def handle(self, bot, update):
```

Comment:
do we use it somewhere?

Reply:
should be fixed by #26, this function wasn't used correctly and that's why we had problems with concurrent threads

Reply:
Gotcha, thanks, closing my PR in favour of yours